tests/libtest.sh

```diff
@@ -529,6 +534,13 @@ os_repository_new_commit ()
     echo "content iteration ${content_iteration}" > usr/bin/content-iteration
 
     ${CMD_PREFIX} ostree --repo=${test_tmpdir}/testos-repo commit  --add-metadata-string "version=${version}" -b $branch -s "Build"
+    if ${CMD_PREFIX} ostree --repo=${test_tmpdir}/testos-repo rev-parse ${branch} 2>/dev/null; then
+        prevdate=$(timestamp_of_commit ${test_tmpdir}/testos-repo "${branch}"^)
+        newdate=$(timestamp_of_commit ${test_tmpdir}/testos-repo "${branch}")
+        if [ $((${prevdate} > ${newdate})) = 1 ]; then
```

Comment:
why not just `if [ $((${prevdate} > ${newdate})) ];` here? according to http://mywiki.wooledge.org/ArithmeticExpression we shouldn't need the `=1` comparison.

Reply:
sorry my comment was late.. it was mostly for my understanding to see if we needed it or not.